### DISCUSSION TITLE
[FR] Formatted(JSON preferable) Response

### DISCUSSION THREAD

**Discussion by mustafayuce33** [pivpn/pivpn#1385] [Feature requests]
**Is your feature request related to a problem? Please describe.**
I use pivpn in our backend services. The responses are in a simple text format. I mean after I run `pivpn -l`, it returns the certificates as a plain text without formatting. So I need to parse this text message to get the meaningful part in it.

**Describe the solution you'd like**
I recommend you to put an argument that we can take the responses in a formatted way, such as JSON. 
Ex; `pivpn -l --json`

**Describe alternatives you've considered**
It would also be great to have a `pivpn nodejs` library to be used in the projects

**Additional context**
No additional context

**Comment by coolapso** [comment]
Please fill in the feature request properly, using the FR template that shows up when you create the "new issue" button.

**Comment by mustafayuce33** [comment]
> Please fill in the feature request properly, using the FR template that shows up when you create the "new issue" button.

Did that.

**Comment by coolapso** [comment]
the json ouput is something that can be considered .. the nodejs library not sure if it something we should build or support, not even sure if we have the know how around ... i know nothing about nodejs other than its name. 

Marked as not priority, but if anyone willing to spend the time with it, ill gladly review it.

**Comment by mustafayuce33** [comment]
Thanks for your kind response
